Weather/weather.ini

```diff
@@ -27,36 +27,48 @@ Information=An minimalistic, still stylish dashboard-like skin with modular comp
 @include=#@#variables.ini
 @include2=#@#include\MeterStyles.inc
 
-; Measure to get the weather data and properly parse it
-[MeasureWeather]
+;FindWeatherLocation
+[FindWeatherLocation]
```

Comment:
This doesn't seem to work properly.

Reply:
Honestly, I'd just guide the user to find their weather code themselves. Otherwise you'd have to search for it every time Rainmeter starts.